Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -174,8 +282,15 @@ void CodeViewWidget::Update()
 
     if (hex_str.length() == VALID_BRANCH_LENGTH && desc != "---")
     {
-      description_item->setText(tr("--> %1").arg(QString::fromStdString(
-          PowerPC::debug_interface.GetDescription(GetBranchFromAddress(addr)))));
+      u32 branch_addr = GetBranchFromAddress(addr);
+      CodeViewBranch& branch = m_branches.emplace_back();
+      branch.src_addr = addr;
+      branch.dst_addr = branch_addr;
+      branch.is_link = ins.ends_with("l") || ins.ends_with("la") || ins.ends_with("l+") ||
```

Comment:
`ends_with` is a C++20 feature. I have feeling at least one of our build bots won't like it.
Consider making a little `IsLink` function anyways to hide away these magic strings.

Reply:
We have `StringEndsWith` in `StringUtil.h` if you want a replacement for `ends_with`.